components/list/list-item-mixin.js

```diff
@@ -55,9 +55,21 @@ export const ListItemMixin = superclass => class extends LocalizeCoreElement(Lis
 			breakpoints: { type: Array },
 			/**
 			 * Whether to render the list-item with reduced whitespace.
+			 * TODO: Remove in favor of padding="slim"
 			 * @type {boolean}
 			 */
 			slim: { type: Boolean },
+			/**
+			 * How much padding to render list items with
+			 * One of 'normal'|'slim'|'none', defaults to 'normal'
+			 * @type {string}
+			 */
+			padding: { type: String },
```

Comment:
Let's call this property `paddingType` / attribute `padding-type` to distinguish it more from the CSS padding property.